packages/bits/package.json

```diff
@@ -142,7 +142,7 @@
     "a11y": "start-server-and-test serve-examples:prod 4200 a11y:base",
     "a11y:base": "yarn run e2e:base --suite a11y",
     "a11y:ci": "yarn run e2e:ci --suite a11y",
-    "assemble": "run-s prettier-check lint build-lib:locl build-lib:prod compodoc-custom-theme build-examples:prod build-schematics",
```

Comment:
Revert changes in package.json (removing prettier-check and lint serves only for making local development faster)